`verboseOutput` `keepOutput` args are not passed to commander

It looks like verbose and output flags are not passed to the commander process.

1) Process logs are guarded:

https://github.com/trevjonez/composer-gradle-plugin/blob/271a5a5ebceb1ea779c800cbcdd1e1a82648f337/commander/os/src/main/kotlin/com/gojuno/commander/os/Processes.kt#L105-L107

2) Logs are turned off by default:

https://github.com/trevjonez/composer-gradle-plugin/blob/271a5a5ebceb1ea779c800cbcdd1e1a82648f337/commander/os/src/main/kotlin/com/gojuno/commander/os/Processes.kt#L29-L32

3) process is unconditionally being called AFAIS:

https://github.com/trevjonez/composer-gradle-plugin/blob/271a5a5ebceb1ea779c800cbcdd1e1a82648f337/commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt#L94-L98

### My use case. 

I am running composer. APK install is failing on the CI and I don't see adb message to get a sense why is it failing. Turning `verboseOutput=true` `keepOutput=true` flags in plugin doesn't help, since apparently flags are not being passed down. So at the end all I see is next:

```
> Task :storage:testDebugComposer
[Tue Feb 25 19:35:37 CET 2020]: Args(appApkPath=/Users/user/Projects/project/storage/build/outputs/apk/androidTest/debug/storage-debug-androidTest.apk, testApkPath=/Users/user/Projects/project/storage/build/outputs/apk/androidTest/debug/storage-debug-androidTest.apk, testRunner=, shard=true, outputDirectory=/Users/user/Projects/project/storage/build/reports/composer/debug, instrumentationArguments=[disableAnalytics, true], verboseOutput=true, keepOutputOnExit=true, devices=[], devicePattern=, installTimeoutSeconds=120, failIfNoTests=true, runWithOrchestrator=false, extraApks=[])
[Tue Feb 25 19:35:37 CET 2020]: 1 connected adb device(s): [AdbDevice(id=emulator-5554, model=Android SDK built for x86_64, online=true)]
[Tue Feb 25 19:35:37 CET 2020]: [emulator-5554] 1 APK to install
[Tue Feb 25 19:35:37 CET 2020]: [emulator-5554] Acquiring lock
[Tue Feb 25 19:35:37 CET 2020]: [emulator-5554] Lock acquired
[Tue Feb 25 19:35:37 CET 2020]: [emulator-5554] Installing apk... pathToApk = /Users/user/Projects/project/storage/build/outputs/apk/androidTest/debug/storage-debug-androidTest.apk
[Tue Feb 25 19:35:47 CET 2020]: [emulator-5554] Failed to install apk /Users/user/Projects/project/storage/build/outputs/apk/androidTest/debug/storage-debug-androidTest.apk
Task :storage:testDebugComposer in storage Finished

> Task :storage:testDebugComposer FAILED
:storage:testDebugComposer (Thread[Execution worker for ':' Thread 5,5,main]) completed. Took 10.469 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':storage:testDebugComposer'.
> Process 'command '/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1
```

Which isn't very helpful since it doesn't contain any error message.

--- a/commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt
+++ b/commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt
@@ -90,11 +90,12 @@ fun connectedAdbDevices(): Single<Set<AdbDevice>> = process(listOf(adb, "devices
 
 fun AdbDevice.log(message: String) = com.gojuno.commander.os.log("[$id] $message")
 
-fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 2 to MINUTES): Observable<Unit> {
+fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 2 to MINUTES, logErrors: Boolean = false): Observable<Unit> {
     val installApk = process(
         commandAndArgs = listOf(adb, "-s", id, "install", "-r", pathToApk),
-        unbufferedOutput = true,
-        timeout = timeout
+        timeout = timeout,
+        print = logErrors,
+        unbufferedOutput = true
     )
 
     return Observable

--- a/composer/composer/src/main/kotlin/com/gojuno/composer/Main.kt
+++ b/composer/composer/src/main/kotlin/com/gojuno/composer/Main.kt
@@ -115,7 +115,7 @@ private fun runAllTests(args: Args, testPackage: TestPackage.Valid, testRunner: 
 
                 val apkPaths = setOf(args.appApkPath, args.testApkPath) + args.extraApks.toSet()
                 val installs = apkPaths.map {
-                    device.installApk(pathToApk = it, timeout = installTimeout)
+                    device.installApk(pathToApk = it, timeout = installTimeout, logErrors = args.verboseOutput)
                 }
 
                 device.log("${installs.size} APK${if (installs.size == 1) "" else "s"} to install")

